.github/workflows/misc-migtests.yml

```diff
@@ -48,7 +48,17 @@ jobs:
 
       - name: "Enable postgres with pg_stat_statements loaded via shared_preload_libraries"
         run: |
-          docker exec ${{ job.services.postgres.id }} sh -c "echo \"shared_preload_libraries = 'pg_stat_statements'\" >> /var/lib/postgresql/data/postgresql.conf"
+          docker exec ${{ job.services.postgres.id }} sh -c "
+            apt-get update && apt-get install -y \
+              postgresql-17-postgis \
+              postgresql-17-postgis-3 \
+              postgresql-contrib-17 \
```

Comment:
Why do we need to install `postgresql-contrib-17` here? Is there any dependency?
Also can you try if the tests work with either `postgresql-17-postgis` or `postgresql-17-postgis-3`, or do we require both.